packages/components/src/confirm-dialog/test/index.js

```diff
@@ -29,6 +29,34 @@ describe( 'Confirm', () => {
 
 				expect( dialog ).toBeInTheDocument();
 
+				elementsTexts.forEach( ( txt ) => {
+					const el = wrapper.getByText( txt );
+					expect( el ).toBeInTheDocument();
+				} );
```

Comment:
I see these are also taken from existing patterns in this test file, but I would recommend doing a `getByRole( 'button', { name: ... } )` query for the buttons.
https://testing-library.com/docs/queries/about/#priority

Reply:
Got it, and love the idea - thanks for the helpful links! pushed 64e34a8